lib/DDG/Fathead/MDNJS.pm

```diff
@@ -4,8 +4,8 @@ use DDG::Fathead;
 
 primary_example_queries "javascript array sort";
 secondary_example_queries
-    "js array sort",
-    "mdn array sort";
+    "js requestanimationframe",
+    "mdn localstorage";
```

Comment:
I wanted to point out that `AudioProcessingEvent` might be interesting but then it turned out it has a wrong abstract (https://mrshu.duckduckgo.com/?q=js+audioprocessingevent&ia=about)

How about `js ClipboardEvent` (https://mrshu.duckduckgo.com/?q=javascript+clipboardevent&ia=about)?